web/beacon-app/src/utils/misc.tsx

```diff
@@ -1,9 +1,15 @@
+import { PATH_DASHBOARD } from '@/application';
 export const StatusIconMap = {};
 
-export const isCurrentMenuPath = (href: string, pathname: string) => {
+export const isCurrentMenuPath = (href: string, pathname: string, href_linked?: string) => {
   const hrefArray = href.split('/');
+  const hrefLinkedArray = href_linked?.split('/') || [];
   const pathnameArray = pathname.split('/');
 
+  if (href_linked === PATH_DASHBOARD.TOPICS && pathnameArray[2] === 'topics') {
```

Comment:
Looks like this change fixed the issue I saw in the video where the effect moved from the bottom of the dashboard menu up to `Projects`. Nice catch!